tests/test_tokenization_common.py

```diff
@@ -53,16 +53,28 @@ def merge_model_tokenizer_mappings(
     return model_tokenizer_mapping
 
 
-class TokenizerTesterMixin:
+DEBUG = False
+
+
+class TokenizerCommonTester(TestCasePlus):
```

Comment:
I think `TokenizerTestCase` or something like that makes more sense here (to indicate it's inherited from `TestCase`)

Reply:
Sure, that works. I just thought of somehow conveying that it's also a container of common tests, and not just "features", as most parent classes are.